MediaGalleryRenditionsApi/Api/ConfigInterface.php

```diff
@@ -6,7 +6,7 @@
 
 declare(strict_types=1);
 
-namespace Magento\MediaGalleryRenditionsApi\Model;
+namespace Magento\MediaGalleryRenditionsApi\Api;
```

Comment:
Th config interface was not meant to be used outside of the renditions module (but can be implemented by extensions). The proper namespace for such interface in the Api module is `Model`. What do you think @sidolov @lenaorobei

Reply:
We need to consider removing the interface and left only implementation marked as api

Reply:
I removed the interface. Did not mark class with `@api` though, since it is patch release.

Reply:
@sivaschenko looks like we don't need `MediaGalleryRenditionsApi ` module for now at all.

Reply:
The interface is needed to be able to provide an alternative implementation.

Reply:
Will move class to `MediaGalleryRenditionsApi/Model`.